src/utils/fixedPoint.ts

```diff
@@ -29,41 +29,42 @@ export function divIfNotZeroUnsafe(
 
 export function bigNumberToString(
   input: BigNumber | null | undefined,
-  decimals: number | number = 2
+  decimalsTarget: number | 2
 ): string {
   if (!input || input === null || input.isZero()) {
     return "0";
   } else if (input.gt("999999999999999999999999999999")) {
     return "∞";
   }
   const inputAsString = Number(ethers.utils.formatEther(input)).toFixed(
-    decimals
+    decimalsTarget
   );
   let outputStr = inputAsString;
-  while (decimals >= 0) {
+  while (decimalsTarget >= 0) {
     if (outputStr.endsWith("0") || outputStr.endsWith(".")) {
       outputStr = outputStr.slice(0, -1);
-    } else decimals = 0;
-    decimals--;
+    } else decimalsTarget = 0;
+    decimalsTarget--;
   }
   return outputStr;
 }
 
 export function fixedNumberToPercentage(
   input: FixedNumber | null | undefined,
-  decimals: number | number = 2
+  decimalsTarget: number | 2,
+  minimumNumberOfDecimals: number | 0
```

Comment:
That's also wrong. Now you're saying it must be either a `number` or `2`, which means that it's ... A `number`.
```suggestion
  decimalsTarget: number = 2,
  minimumNumberOfDecimals: number = 0
```
Also consider namings `targetDecimals` and `minDecimalCount`.